src/discord.coffee

```diff
@@ -18,10 +18,27 @@ catch
 
 Discord             = require "discord.js"
 TextChannel         = Discord.TextChannel
+ReactionMessage     = require "./reaction_message"
```

Comment:
Classes should be title case

Reply:
The class name is title case.  From what I've read, many javascript libraries still use all lowercase for filenames.  I think this should work.

Reply:
javascript libraries that export classes use title case for the file, and camel if it exports functions